packages/cashier/src/pages/account-transfer/account-transfer-locked/__tests__/account-transfer-locked.spec.tsx

```diff
@@ -41,10 +41,12 @@ describe('AccountTransferLocked', () => {
             {
                 content: 'Complete the financial assessment form',
                 status: 'action',
+                is_disabled: false,
+                button_text: 'sample text',
                 onClick,
             },
         ];
-        render(<Checklist className='cashier-locked__checklist' items={items} />);
+        render(<Checklist className='cashier-locked__checklist' itemClassName='sample_class_name' items={items} />);
         const btn = screen.getByTestId('dt_checklist_item_status_action');
```

Comment:
should we define types for such constants?
 as I understand, typescript can know, when we are getting some dom element by its tag (for example, querySelector('a'). but in this case TS don’t know the type
correct me please

Reply:
actually here TS infers it as `HTMLElement`